test.disabled/ejabberd_tests/tests/login_SUITE.erl

```diff
@@ -404,7 +404,12 @@ log_one_scram(Config) ->
 
 
 log_non_existent_plain(Config) ->
-    {auth_failed, _, _} = log_non_existent(Config).
+    {auth_failed, _, Xmlel} = log_non_existent(Config),
+    {xmlel, <<"failure">>, _Attrs, Data} = Xmlel,
+    true = lists:any(
```

Comment:
Please use `exml_query` wherever it is reasonable to do so. This call can be replaced with `#xmlel{} = exml_query:subelement(Xmlel, <<"not-authorized">>).`